modules/ui/sidebar.js

```diff
@@ -412,6 +418,40 @@ export function uiSidebar(context) {
                 });
         };
 
+        /*
+            The checkWindowWidth function checks the width of the working window when the mouse is over the
+            working area and flagCollapse and flagExpand is to prevent the re-triggering action after the window
+            is resized if the window is resized and the flags are not used then whenever we will try to expand
+            the side bar when the window width is below 700 then it will automatically collapse the side bar
+            which is not what we want.
+        */ 
+            function checkWindowWidth()
```

Comment:
Why is this entire function indented in relation to the comment above (and surrounding code)? IIUC it should have 4 fewer spaces of indentation.